resources/fetchers/process_fetcher_test.go

```diff
@@ -0,0 +1,184 @@
+package fetchers
+
+import (
+	"context"
+	"fmt"
+	"github.com/elastic/cloudbeat/resources/fetching"
+	"github.com/mitchellh/mapstructure"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/yaml.v2"
+	"io/fs"
+	"k8s.io/apimachinery/pkg/util/json"
+	"testing"
+	"testing/fstest"
+)
+
+const (
+	statContent = `1167 (containerd-shim) S 1 1167 198 0 -1 1077952768 223005 9831 39 0 665 1329 8 10 20 0 12 0 76222 730476544 2268 18446744073709551615 1 1 0 0 0 0 1006249984 0 2143420159 0 0 0 17 2 0 0 0 0 0 0 0 0 0 0 0 0 0`
+)
+
+type TextProcessContext struct {
+	Pid               string
+	Name              string
+	ConfigFileFlagKey string
+	ConfigFilePath    string
+}
+
+type ProcessConfigTestStruct struct {
+	A string
+	B int
+}
+
+var status = `Name:   %s`
+var cmdline = `/usr/bin/%s --kubeconfig=/etc/kubernetes/kubelet.conf --%s=%s`
+
+func TestFetchWhenFlagExistsButNoFile(t *testing.T) {
+	testProcess := TextProcessContext{
+		Pid:               "3",
+		Name:              "kubelet",
+		ConfigFileFlagKey: "fetcherConfig",
+		ConfigFilePath:    "test/path",
+	}
+	sysfs := createProcess(testProcess)
+
+	fetcherConfig := ProcessFetcherConfig{
+		BaseFetcherConfig: fetching.BaseFetcherConfig{},
+		RequiredProcesses: map[string]ProcessInputConfiguration{
+			"kubelet": {CommandArguments: []string{"fetcherConfig"}}},
+		Fs: sysfs,
+	}
+	processesFetcher := &ProcessesFetcher{cfg: fetcherConfig}
+
+	fetchedResource, err := processesFetcher.Fetch(context.TODO())
+	assert.Nil(t, err)
+	assert.Equal(t, len(fetchedResource), 1)
```

Comment:
When asserting the first argument is the `expected` and the second is the `actual`
Please make sure all over the pr that you are aligned with that

Reply:
opppppp, I saw it during my self review and I forgot to fix it - will do